harness/determined/common/schemas/expconf/_gen.py

```diff
@@ -797,6 +797,7 @@
         "image": {
             "type": [
                 "object",
+                "string",
```

Comment:
I assume you're going to rebase away from that schema change.

Reply:
oops, will fix